test/fixtures/testArrayCompletionSchema.json

```diff
@@ -1,85 +1,110 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema#",
-    "definitions": {
-      "obj1": {
-        "properties": {
-          "obj1": {
-            "type": "object"
-          }
-        },
-        "required": [
-          "obj1"
-        ],
-        "type": "object"
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
the changes aren't very readable inside the github diff tool:
 - I just added a new test schema `test_array_nested_anyOf`
 - add `obj3` into the `definitions`
 - autoformat file (there were two extra spaces)